docker/signer-dmz/scripts/jwks_to_pem.py

```
@@ -71,21 +71,39 @@ def main() -> int:
+        # Local / Docker Desktop: DMZ must load the same JWKS the app serves; TLS is often absent.
+        pass
+    else:
         print(
-            f"jwks_to_pem: refusing to fetch JWKS from non-https URL: {args.url!r}",
+            f"jwks_to_pem: JWKS URL must be https, or http on localhost/127.0.0.1/host.docker.internal: {args.url!r}",
             file=sys.stderr,
         )
         return 1
 
-    ctx = ssl.create_default_context()
     req = urllib.request.Request(args.url, headers={"User-Agent": "pymthouse-signer-dmz/1.0"})
```

Ruff (0.15.10)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)